storage/iam.rb

```diff
@@ -109,6 +109,49 @@ def add_bucket_conditional_iam_binding bucket_name:, role:, member:, title:, des
   # [END storage_add_bucket_conditional_iam_binding]
 end
 
+def remove_bucket_conditional_iam_binding bucket_name:, role:, title:, description:, expression:
+  # [START storage_remove_bucket_conditional_iam_binding]
+  # bucket_name = "Your Google Cloud Storage bucket name"
+  # role        = "Bucket-level IAM role"
+  # title       = "Condition Title"
+  # description = "Condition Description"
+  # expression  = "Condition Expression"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new
+  bucket = storage.bucket bucket_name
+
+  binding_to_remove = nil
+  bucket.policy requested_policy_version: 3 do |policy|
+    policy.version = 3
+
+    policy.bindings.each do |b|
+      condition = {
+        title:       title,
+        description: description,
+        expression:  expression
+      }
+      if (b.role == role) && (b.condition &&
+        b.condition.title == title &&
+        b.condition.description == description &&
+        b.condition.expression == expression)
+        binding_to_remove = b
+      end
+    end
+    if binding_to_remove
```

Comment:
I would just merge the removal and `puts`:
```ruby
if binding_to_remove
  policy.bindings.remove binding_to_remove
  puts "Conditional Binding was removed."
else
  puts "No matching conditional binding found."
end
```

Reply:
done!